src/constants/apiPermissions.js

```diff
@@ -1,8 +1,18 @@
-// const roles = require('./roles')
+const roles = require('./roles')
 
 module.exports = {
   User: {
-
+    getAll: [roles.SUPER],
+    getById: [ roles.SUPER, roles.OWNER ],
+    create: [ roles.SUPER ],
+    update: [ roles.ADMIN, roles.OWNER ],
```

Comment:
check the [spec](https://github.com/foundersandcoders/open-tourism-platform/blob/master/data_spec_v1.md#users), it says only owner or super. What do you think?

Reply:
tbh I wrote these a few weeks ago, but I think my logic is that we probably want to allow admins to be able change the name/organisation name and stuff, but only SUPER can do username password and role - I think this is ok?

Reply:
yeh sure